addons/packages/packetfence.spec

```diff
@@ -583,6 +583,10 @@ if [ ! -f /usr/local/pf/conf/ssl/server.crt ]; then
     cat /usr/local/pf/conf/ssl/server.crt /usr/local/pf/conf/ssl/server.key > /usr/local/pf/conf/ssl/server.pem
 fi
 
+# Create OMAPI key
+if [ ! -f /usr/local/pf/conf/pf_omapi_key ]; then
+    echo $(date) $HOSTNAME | base64 > /usr/local/pf/conf/pf_omapi_key
```

Comment:
Something like that:
dd if=/dev/urandom bs=16 count=1 2>/dev/null | openssl enc -e -base64